.travis.yml

```diff
@@ -1,51 +1,48 @@
 language: node_js
+os: linux
 node_js:
-    - '14'
+  - '14'
 
 env:
-    # skip Puppeteer download, only needed for Netlify build
-    - PUPPETEER_SKIP_DOWNLOAD=1
+  # skip Puppeteer download, only needed for Netlify build
+  - PUPPETEER_SKIP_DOWNLOAD=1
 
 addons:
-    apt:
-        packages:
-            - libgconf-2-4
+  apt:
+    packages:
+      - libgconf-2-4
 
 cache:
-    yarn: true
-    directories:
-        - 'node_modules'
-        - '~/.cache'
+  yarn: true
+  directories:
+    - 'node_modules'
+    - '~/.cache'
 
 before_script:
-    - yarn global add license-checker
-    - yarn run lerna bootstrap
-    - git checkout .
-    - yarn run cy:verify
-    - yarn run cy:info
+  - yarn global add license-checker
+  - yarn run lerna bootstrap
+  - git checkout .
+  - yarn run cy:verify
+  - yarn run cy:info
 
 script:
-    - yarn run check-licenses
-    - yarn run dependency-cruiser
-    - yarn run lint
-    - yarn test
-    # All Percy snapshots except SearchBar have been moved to Cypress, to avoid confusion disable the old "snapshot" command.
-    # When the Cypress setuisbe verified, this command might be removed.
-    # - yarn run snapshot
-    - yarn run cy:run --record --key ${CYPRESS_RECORD_KEY}
-    # after all tests finish running we need
-    # to kill all background jobs (like "npm start &")
-    # this avoids flake in Travis jobs
-    - kill $(jobs -p) || true
+  - yarn run check-licenses
+  - yarn run dependency-cruiser
+  - yarn run lint
+  - yarn test
+  # All Percy snapshots except SearchBar have been moved to Cypress, to avoid confusion disable the old "snapshot" command.
+  # When the Cypress setuisbe verified, this command might be removed.
+  # - yarn run snapshot
+  - yarn run cy:run --record --key ${CYPRESS_RECORD_KEY}
 
 before_deploy:
-    - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc
+  - echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" >> $HOME/.npmrc
 
 deploy:
-    if: branch = master AND tag IS present
-    provider: script
+  - provider: script
     script: 'yarn run lerna:publish'
+    cleanup: false
     skip_cleanup: true
     on:
-        tags: true
```

Comment:
@pshoniuk I believe the problem is that tags don't run on `branch: master`, as you can see in the [previous commit](https://github.com/Giphy/giphy-js/commit/5f72f0c752728f2cb96fb3442be7421229667c7b) I had this setting before trying `tags: true` 

Unless you changed something else I missed, or Travis behavior has changed, this probably won't work